module/apmfiber/go.mod

```
@@ -0,0 +1,19 @@
+module go.elastic.co/apm/module/apmfiber
+
+require (
+	github.com/gofiber/fiber/v2 v2.18.0
+	github.com/pkg/errors v0.9.1
+	github.com/stretchr/testify v1.7.0
```

```suggestion
	github.com/stretchr/testify v1.6.1
```

It appears testify v1.7.0 does not work with Go 1.12: https://apm-ci.elastic.co/blue/organizations/jenkins/apm-agent-go%2Fapm-agent-go-mbp/detail/PR-999/30/pipeline/129

Alternatively, I think it would be reasonable to require a more recent version of Go for this module.

---

agree, raised the version of go in the tag